endpoints/getting-started-grpc/README.md

```diff
@@ -44,11 +44,10 @@ $ protoc \
 
 1. In `api_config.yaml`, replace `MY_PROJECT_ID` and `SERVICE-ACCOUNT-ID` with your Project ID and your service account's email address respectively.
 
-1. Deploy your service's configuration to Endpoints. Take note of your service's config ID and name once the deployment completes.
+1. Deploy your service's configuration to Endpoints. 
```

Comment:
We should still say:
Take note of your service's name once the deployment completes.